glusterd2/commands/volumes/volume-create-txn.go

```diff
@@ -189,15 +189,9 @@ func createVolinfo(c transaction.TxnCtx) error {
 		return err
 	}
 
-	// TODO: Expose this granularity in the volume create API.
-	var checks brick.InitChecks
-	if !req.Force {
-		checks.IsInUse = true
-		checks.IsMount = true
-		checks.IsOnRoot = true
-	}
+	checks := brick.NewInitCheck().PrepareChecks(req.Force, req.Flags)
```

Comment:
This looks complicated. You can pass just `req to `NewInitCheck()` function.

Reply:
done